hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommits.java

```diff
@@ -287,30 +288,29 @@ public void testThriftExceptionConcurrentCommit() throws TException, Interrupted
     spyOps.commit(metadataV2, metadataV1);
 
     ops.refresh();
-    Assert.assertNotEquals("Current metadata should have changed", metadataV2, ops.current());
-    Assert.assertTrue(
-        "Current metadata file should still exist", metadataFileExists(ops.current()));
-    Assert.assertEquals(
-        "The column addition from the concurrent commit should have been successful",
-        2,
-        ops.current().schema().columns().size());
+    Assertions.assertThat(ops.current()).as("Current metadata should have changed").isNotEqualTo(metadataV2);
+    Assertions.assertThat(
+        metadataFileExists(ops.current())).as( "Current metadata file should still exist").isTrue();
+    Assertions.assertThat(
+        ops.current().schema().columns()).as(
+        "The column addition from the concurrent commit should have been successful").hasSize(  2);
   }
 
   @Test
   public void testInvalidObjectException() {
     TableIdentifier badTi = TableIdentifier.of(DB_NAME, "`tbl`");
-    Assert.assertThrows(
-        String.format("Invalid table name for %s.%s", DB_NAME, "`tbl`"),
-        ValidationException.class,
-        () -> catalog.createTable(badTi, schema, PartitionSpec.unpartitioned()));
+    Assertions.assertThatThrownBy( () -> catalog.createTable(badTi, schema, PartitionSpec.unpartitioned()),
+            String.format("Invalid table name for %s.%s", DB_NAME, "`tbl`"),
+        ValidationException.class
+        );
   }
 
   @Test
   public void testAlreadyExistsException() {
-    Assert.assertThrows(
-        String.format("Table already exists: %s.%s", DB_NAME, TABLE_NAME),
-        AlreadyExistsException.class,
-        () -> catalog.createTable(TABLE_IDENTIFIER, schema, PartitionSpec.unpartitioned()));
+    Assertions.assertThatThrownBy(() -> catalog.createTable(TABLE_IDENTIFIER, schema, PartitionSpec.unpartitioned()),
```

Comment:
this should be of the form `assertThatThrownBy(...).isInstanceOf(AlreadyExistsException.class).hasMessage(...)`. Same for the other places that are being changed

Reply:
Modified as suggested